rust/main/chains/hyperlane-ethereum/src/contracts/mailbox.rs

```diff
@@ -611,32 +611,21 @@ where
                 )
             })
             .collect::<ChainResult<Vec<_>>>()?;
-        if let Some(contract_call) = contract_calls.first() {
-            self.refresh_block_and_fee_cache(&contract_call.tx).await?;
-        }
-        let filled_tx_params_futures = contract_calls.iter().map(|tx| {
-            fill_tx_gas_params(
-                tx.clone(),
-                self.provider.clone(),
-                &self.conn.transaction_overrides,
-                &self.domain,
-                true,
-                self.cache.clone(),
-            )
-        });
-        let contract_calls = join_all(filled_tx_params_futures)
-            .await
-            .into_iter()
-            .collect::<ChainResult<Vec<_>>>()?;
```

Comment:
Since we removed population of tx params on individual calls, calculation of [individual calls gas limit aggregate](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/0c47163281fa5a9381f4a38fb5b6995c8e9a382d/rust/main/chains/hyperlane-ethereum/src/contracts/multicall.rs#L78) will be irrelevant.

Reply:
We will bring the aggregation of gas of individual calls in this PR - https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/5974